CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+<a name="2.0.0-alpha.8"></a>
+# [2.0.0-alpha.8 ectoplasm-helicopter](https://github.com/angular/material2/compare/g3_v2_0...2.0.0-alpha.8) (2016-09-01)
+
+### BREAKING CHANGES
+
+* all: we've updated our packaging to match angular/angular's packaging.  If you're using System.js in your project, you will probably want to
+switch to using our UMD bundles.  Example config:
+
+```ts
+'@angular2-material/core': {
+    format: 'cjs',
+    main: 'core.umd.js'
+  }
+```
+
+You can see a more detailed example in our demo app's system config [here](https://github.com/angular/material2/blob/master/src/demo-app/system-config.ts).
+
+* all: material modules must be included with `forRoot()` now so that they work properly with lazy loading:
```

Comment:
Oh, I guess that's confusing. I meant we need both options so lazy loading will work *without*. Probably best to leave that out.